src/state/opamSwitchState.ml

```diff
@@ -1078,7 +1078,7 @@ let unavailable_reason st ?(default="") (name, vformula) =
     | Some missing ->
       Printf.sprintf
         "depends on the unavailable system package '%s'. Use \
-         `--assume-depexts' to attempt installation anyway."
+         `--no-depexts' to attempt installation anyway."
```

Comment:
Thoughts from this morning's dev meeting: it would be worth informing the user that the issue _might_ be the opam file itself (i.e. a mis-typed depext), so perhaps something like:

```suggestion
         `--no-depexts' to attempt installation anyway, or it is \
         possible that a depext package name in the opam file \
         is incorrect."
```